src/sparseml/exporters/transforms/utils/matching.py

```diff
@@ -337,7 +337,13 @@ def _match_children(
     #       is the graph output.
     # this is a difference in behavior to get_node_parents, which replaces input
     # with None, instead of removing it.
-    if not (len(children_ops) <= len(children)):
+    non_optional_children_ops = [
```

Comment:
@corey-nm opinion?

Reply:
Is this to handle cases where the children are all optional? I added something similar in another PR cause I ran into this case when testing as well 😁

Reply:
as discussed -> your PR solves exactly this problem. will remove this hack before landing